app/src/main/java/se/nullable/flickboard/ui/voice/GetVoiceInputId.kt

```diff
@@ -0,0 +1,14 @@
+package se.nullable.flickboard.ui.voice
```

Comment:
Thinking on it I imagine this should be in Util instead of UI following the refactor?

Reply:
Yeah.

Reply:
Minor thing, but please format variables as `foo: Bar`, not `foo : Bar`. I'd suggest enabling Android Studio's "format/organize on commit" feature, which should catch this stuff.